python/cuml/cuml/tests/test_kneighbors_classifier.py

```diff
@@ -218,6 +218,8 @@ def test_predict_large_n_classes(datatype):
     assert array_equal(y_hat.astype(np.int32), y_test.astype(np.int32))
 
 
+# Ignore FutureWarning: Using `__dataframe__` is deprecated
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
This makes sure we only ignore the dataframe warning and not all `FutureWarnings`

```suggestion
@pytest.mark.filterwarnings("ignore:Support for loading dataframes via the `__dataframe__` interchange protocol is deprecated")
```

(same for the other occurrence)

Reply:
I'm unable to reproduce getting a warning in this test (and don't see how one could be generated). I _think_ this one can just be dropped.

Reply:
@jcrist What version of cudf are you using? Only recent `25.02` nightlies will show this.

Reply:
`25.02.00a273`. I see the warning at the other location, but not here.

Reply:
I cannot reproduce this one either.

Reply:
I filed https://github.com/rapidsai/cuml/pull/6239 as a follow-up with this filter removed. My hope is that this PR passes CI and we can merge it as-is, then follow up with that PR.